examples/docker/compose/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.8'
+
+services:
+  gpu-app1:
+    image: nvidia/cuda:11.5.2-runtime-ubuntu20.04
+    command: nvidia-smi
```

Comment:
I added `-l 1` to here and L17 for nvidia-smi to loop forever. 

It appears both containers print the same GPU ID. Note the SkyPilot task has 2 GPUs assigned, so GPUs 0 and 1 are available.

Is there any env var (CUDA_VISIBLE_DEVICES?) we can add to this file to show how to distribute the containers to GPUs 0 and 1 respectively? Can even be a comment.

Reply:
Good point - I've changed from `count` to explicit `device_id`. Note that nvidia-docker remaps device ids, so from within `gpu-app2` container the GPU ID visible will be ['0'] (though it maps to physical device 1). Also added this as a comment.